src/contrib/php_array_api.h

```diff
@@ -25,19 +25,11 @@
 #include "zend_hash.h"
 #include "zend_list.h"
 
-#ifdef ZEND_ENGINE_3
```

Comment:
`php_array_api.h` is vendored from https://github.com/sgolemon/php-array-api, so I'm not too concerned with this code lingering. If you want to contribute these changes upstream first (since this isn't an urgent bug fix), that would allow us to keep things in sync. Sara may also want to make a tag before removing PHP 5.x macros.

Reply:
In that case, we can keep it in for the time being. Closing this PR in favour of updating the file later.